app/src/main/java/com/example/movopfy/features/home/data/api/anilibria/AnilibriaURL.kt

```diff
@@ -5,6 +5,6 @@ import com.example.movopfy.retrofit.RetrofitClient
 class AnilibriaURL(private val retrofitClient: RetrofitClient) {
```

Comment:
That class should be removed. Will be disscused on a call

Reply:
I did